File: 준영/프로그래머스/LV2/숫자의 표현.py
def solution(n):
    answer = 0
    for i in range(1, n+1):
        temp = i
        sum_a = i
        while sum_a < n:
            temp += 1
            sum_a += temp
        if sum_a == n:
            answer += 1
    
    return answer

solution(15)

File: 준영/프로그래머스/LV2/숫자의 표현.js
function solution(n) {
  var answer = 0
  for (let i = 1; i < n+1; i++) {
    var temp = i
    var sum_a = i
    while (sum_a < n) {
      temp += 1
      sum_a += temp
    }
    if (sum_a === n) {
      answer += 1
    }
    
  }
  return answer
}